src/content/docs/style-guide/word-choice/usage-dictionary.mdx

```diff
@@ -928,8 +928,8 @@ For more information about observability, see [Product and cabilities: observabi
   >
     Docs use the [preview callout](/docs/style-guide/quick-reference/callouts/) for features in [preview](#new-preview). In the body text, use lowercase. For example:
 
-    <Callout title="PREVIEW FEATURE">
-      This feature is currently in preview.
+    <Callout title="preview">
+        We're still working on this feature, but we'd love for you to try it out!
     </Callout>
 
     If the developer team prefers to use a term other than preview, clarify what is driving that use of the term (Legal requirement?), and add any relevant info here in the usage dictionary.
```

Comment:
@nbaenam since legal has told us to use this format, I think you could remove this line. It opens the door to changes that don't follow the New Relic policy.

How about just adding a line here like this: In addition to your invitation to the preview, you should add boilerplate legal language that's mentioned in our [style guide](/docs/style-guide/structure/callouts).

Reply:
Hi @rhetoric101, good catch! I've rewritten it a little bit:
`In addition to your preview invitation, you should add legal boilerplate language as outlined in our [style guide](/docs/style-guide/structure/callouts/#preview-callouts).`